src/test/java/com/wikia/webdriver/common/contentpatterns/AdsContent.java

```diff
@@ -47,7 +48,11 @@ private AdsContent() {
 
   }
 
-  public static Map<String, String> getSlotsSelectorsMap() {
+  public static Map<String, String> getSlotsSelectorsMap(String env) {
+    if(env.equals("mobile")) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Move the "mobile" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1132)